### PR TITLE
Course create from template visibility help

### PR DESCRIPTION
**Pull request by Termanty**

### CODE REVIEW

File: app/views/course_templates/index.html.erb [hunk 1]
```diff
@@ -20,13 +21,21 @@
         <td><%= course_template.description %></td>
         <td><%= course_template.material_url %></td>
         <td><%= course_template.source_url %></td>
+        <td><%= course_template.expires_at %></td>
         <td><%= link_to 'Edit', edit_course_template_path(course_template) %></td>
         <td><%= link_to 'Destroy', course_template, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+        <td><% if course_template.hidden %>
```
Comment (jamox) [inline app/views/course_templates/index.html.erb]: Maybe move the if to its own line

Comment (jsopakar) [inline app/views/course_templates/index.html.erb]: Fixed

File: config/routes.rb [hunk 2]
```diff
@@ -1,10 +1,12 @@
 TmcServer::Application.routes.draw do
+  get 'org/:organization_id/course_templates', to: 'course_templates#list_for_teachers', as: 'list_course_templates'
+  get 'org/:organization_id/course_templates/:id', to: 'course_templates#prepare_course', as: 'prepare_course'
```
Comment (jamox) [inline config/routes.rb]: not a big fan of manual matchers; we probably could turn these into members of the ` resources :organizations, path: 'org' do`

File: app/controllers/courses_controller.rb [hunk 3]
```diff
@@ -82,13 +75,19 @@ def create
...
+        refresh_course(@course)
+        format.html { redirect_to(organization_course_help_path(@organization, @course), notice: 'Course was successfully created.') }
       else
         format.html { render action: 'new', notice: 'Course could not be created.' }
       end
     end
   end
 
+  def help
+    @course = Course.find(params[:course_id])
```
Comment (jamox) [inline app/controllers/courses_controller.rb]: use the prefore action (you'll see it after rebase)

**Comment by jamox** [comment]
Tried merging and some tests were failing, maybe due to unsuccessful merge... 
Please rebase with the testmycode/tmc-server/tmc-teacher-tools so that it will merge clean.

**Pull request closed without merging**